data/sql/updates/pending_db_world/rev_1609803806703890000.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1609803806703890000');
+
+-- add initial delay of half a second to IC Update - Cast Spell Lightning Shield
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 17982) AND (`source_type` = 0) AND (`id` IN (4));
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(17982, 0, 4, 0, 0, 0, 100, 0, 500, 500, 15000, 15000, 0, 11, 325, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'IC Update - Cast Spell Lightning Shield');
```

Comment:
You dont use comments from Keira? Missing NPC name

Reply:
:O